src/objects/core/admin.py

```diff
@@ -13,6 +18,32 @@ class ObjectTypeAdmin(admin.ModelAdmin):
     )
     readonly_fields = ("_name",)
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "<int:objecttype_id>/versions/",
```

Comment:
Since it's for internal use only, let's prefix it with an underscore
```suggestion
                "<int:objecttype_id>/_versions/",
```